clients/es-integration/src/integration-test/java/com/github/ymikevich/es/integration/services/DefaultTweetServiceIT.java

```diff
@@ -0,0 +1,223 @@
+            + "      \"text\": \"Hello Hey\",\n"
+            + "      \"source\": \"http://somesource.com\",\n"
+            + "      \"favoriteCount\": 100,\n"
+            + "      \"retweetCount\": 150,\n"
+            + "      \"user\": {\n"
+            + "        \"id\": \"10\",\n"
+            + "        \"email\": \"bred@mail.com\",\n"
+            + "        \"name\": \"humortranslate\"\n"
+            + "      }\n"
+            + "    },\n"
+            + "    {\n"
+            + "      \"tweetId\": 125324235,\n"
+            + "      \"text\": \"Hey\",\n"
+            + "      \"source\": \"http://somesource.com\",\n"
+            + "      \"favoriteCount\": 250,\n"
+            + "      \"retweetCount\": 300,\n"
+            + "      \"user\": {\n"
+            + "        \"id\": \"10\",\n"
+            + "        \"email\": \"bred@mail.com\",\n"
+            + "        \"name\": \"humortranslate\"\n"
+            + "      }\n"
+            + "    }\n"
+            + "  ],\n"
+            + "  \"pagination\": {\n"
+            + "    \"page\": 0,\n"
+            + "    \"count\": 10\n"
+            + "  }\n"
+            + "}";
+
+    //language=JSON
+    private static final String JSON_SEARCH_REQUEST_WITH_PAGINATION = "{\n"
+            + "  \"searchString\": \"Hey\",\n"
+            + "  \"pagination\": {\n"
+            + "  \t\"page\": 0,\n"
+            + "  \t\"count\": 1\n"
+            + "  },\n"
+            + "  \"sorting\":{\n"
+            + "    \"fieldName\": \"retweetCount\",\n"
+            + "    \"direction\": \"ASC\"\n"
+            + "  }\n"
+            + "}";
+
+    //language=JSON
+    private static final String EXPECTED_JSON_SEARCH_RESPONSE_WITH_PAGINATION = "{\n" +
+            "  \"responseEntities\": [\n"
+            + "    {\n"
+            + "      \"tweetId\": 265473423,\n"
+            + "      \"text\": \"Hello Hey\",\n"
+            + "      \"source\": \"http://somesource.com\",\n"
+            + "      \"favoriteCount\": 100,\n"
+            + "      \"retweetCount\": 150,\n"
+            + "      \"user\": {\n"
+            + "        \"id\": \"10\",\n"
+            + "        \"email\": \"bred@mail.com\",\n"
+            + "        \"name\": \"humortranslate\"\n"
+            + "      }\n"
+            + "    }\n"
+            + "  ],\n"
+            + "  \"pagination\": {\n"
+            + "    \"page\": 0,\n"
+            + "    \"count\": 1\n"
+            + "  }\n"
+            + "}";
+
+    private static final String EXPECTED_JSON_STATISTICS_RESPONSE = "{\n"
+            + "  \"averageLikeValue\": 283.3333333333333,\n"
+            + "  \"mostRetweetedUserTweet\": {\n"
+            + "    \"retweetValue\": 500,\n"
+            + "    \"tweetId\": \"375685667\",\n"
+            + "    \"source\": \"http://somesource.com\"\n"
+            + "  }\n"
+            + "}";
+
+
+    @Before
+    public void setUp() {
+        mockMvc = MockMvcBuilders.webAppContextSetup(context)
```

Comment:
I think you're doing the same job here as spring could do for you by configuring MockMvc with standard context